packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -43,14 +44,10 @@ export interface CppBuildConfiguration {
 }
 
 /**
- * Representation of a saved build configuration in local storage.
+ * Representation of all saved build configurations per workspace root in local storage.
  */
-class SavedActiveBuildConfiguration {
-
-    /**
-     * The name of the build configuration.
-     */
-    configName?: string;
+class SavedActiveBuildConfigurations {
+    configs: [string, CppBuildConfiguration | undefined][];
```

Comment:
I would find it more natural if this was a map,

```
{
  "configs": {
    "/tmp": "/tmp/build/release"
  }
}
```

Is there a reason not to do that?